operator/cmd/manager/main.go

```diff
@@ -117,11 +133,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = ensureWebhookConfigVolume(cfg, namespace); err != nil {
+		log.Error(err, "Failed to ensure webhook volume")
+	}
+	if err = ensureWebhookCertificate(cfg, namespace); err != nil {
+		log.Error(err, "Failed to ensure webhook CA configuration")
```

Comment:
do we want to os.Exit(1) here like a lot of the scaffolded code does? and above too

Reply:
I considered this but opted against it. If for some reason we are not successful here, the operator can still function effectively for many use cases.

I do think we should consider exposing some sort of health metric that we could put these kinds of exceptions on.